src/assets/dotnet.md

```diff
@@ -1,5 +1,54 @@
 ## .NET
 ---
 ### Prerequisites
+In order to use Allure in your .Net project with the following exemple you will need used NUnit with NuGet NUnit.Allure
```

Comment:
I have not understood. Is it NUnit and NUnit.Allure from NuGet?

Reply:
A NuGet is a package manager for .NET.
NUnit is a unit-testing framework for all .Net languages. It is necessary to install a NuGet to use it. On the other hand, NUnit.Allure is a NuGet to use the Allure tags.

Reply:
"I have not understood it" was a way to say that probably more information would be useful.